src/main/java/frc/robot/RobotId.java

```diff
@@ -28,17 +34,83 @@ public enum RobotId {
         this.subsystems = List.of(subsystems);
     }
 
+    /**
+     * @Deprecated The RobotId enum should just be for robot names
+     * @return
+     */
+    @Deprecated
     public List<Class<? extends SubsystemBase>> getSubsystems() {
         return subsystems;
     }
 
+    /**
+     * Is this robot a classbot?
+     * @return true if a classbot
+     * @deprecated this method is not needed....
+     */
+    @Deprecated
```

Comment:
Why is this deprecated? It could be useful if we want to only do something if it's a classbot without checking all 4 possibilities.

Reply:
1. I do not believe that this predicate should be used. The characterizations should be done in RobotContainer when the robot is built/declared.
2. Deprecated does not prevent someone from using the predicate. It should make them think twice about using it.
3. The classbots are moribund. IIRC, only one is in working condition.